<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualControl</class>
 <widget class="QDialog" name="ManualControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ручное управление</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="coordinateBox">
     <property name="title">
      <string>Координаты</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutX">
        <item>
         <widget class="QLabel" name="coordinateXLabel">
          <property name="text">
           <string>Координата X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="coordinateXEdit">
          <property name="inputMask">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutY">
        <item>
         <widget class="QLabel" name="coordinateYLabel">
          <property name="text">
           <string>Координата Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="coordinateYEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutZ">
        <item>
         <widget class="QLabel" name="coordinateZLabel">
          <property name="text">
           <string>Координата Z</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="coordinateZEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutF">
        <item>
         <widget class="QLabel" name="coordinateFLabel">
          <property name="text">
           <string>Координата F</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="coordinateFEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Reset</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>buttonBox</zorder>
  <zorder>coordinateBox</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ManualControl</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ManualControl</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
